qemu_v8.mk

```diff
@@ -65,7 +65,9 @@ arm-tf-clean:
 # QEMU
 ################################################################################
 qemu:
-	cd $(QEMU_PATH); ./configure --target-list=aarch64-softmmu\
+	cd $(QEMU_PATH); \
+		git submodule update --init dtc && \
```

Comment:
@jbech-linaro 
I'll probably agree with Jerome,
It could be done by adding default attribute to the manifest (based on info from [manifest-format.txt](https://gerrit.googlesource.com/git-repo/+/master/docs/manifest-format.txt#171))
```
<default sync-s="true"/>
```
And to sync git reps + all submodules:
```
repo sync --fetch-submodules
```